config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -124,6 +125,8 @@ presubmits:
           limits:
             cpu: 4
             memory: 14Gi
+        securityContext:
+          privileged: true
         command:
         - wrapper.sh
```

Comment:
please migrate non-kind jobs off of krte at some point, so we can continue to iterate on this image more tightly

Reply:
ACK, currently they are using krte only for dind, go and to ensure we aren't somehow dependent on bootstrap (via kubekins) (atleast to start with).
Eventually, will move to a kubekins(esque) image with kubetest2.